src/loader-components/_Badge.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-import styles from './NpmBadge.module.scss';
+import styles from './Badge.module.scss';
```

Comment:
Lets pluralize this file and the css, `Badges`

Reply:
Changed here : https://github.com/cerner/terra-dev-site/pull/243/commits/6dde4584e4c4d54343e4b4f2efdf3f0be133ebf7